src/components/Requirements/IncompleteSubReqCourse.vue

```diff
@@ -15,37 +15,36 @@
       <draggable
         class="draggable-requirements-courses"
         :group="{ name: 'draggable-semester-courses', put: false }"
-        :value="courses"
+        :modelValue="courses"
```

Comment:
Would it be better to use `v-model` here?

![Screen Shot 2021-04-08 at 9 21 48 AM](https://user-images.githubusercontent.com/54298311/114034067-dba62500-984b-11eb-8f01-0e8ecb0d9f51.png)

Reply:
To ensure that it's readonly.